custom-metrics-stackdriver-adapter/examples/prometheus-to-sd/Makefile

```diff
@@ -0,0 +1,16 @@
+TAG = v0.1.0
+PREFIX = gcr.io/bskiba-dev
```

Comment:
Maybe change this to something less personal? e.g. gcr.io/google-containers?

Reply:
Or maybe an env variable? I'd rather avoid people randomly trying to push to google-containers.

Reply:
Actually this image is also used in example deployment file, so I think your intention was to actually push this to a publicly accessible registry? We have Makefiles with default prefix of gcr.io/google-containers everywhere, I think we have issue in general that it's too easy to accidentally ovewrite production images. I wouldn't worry about it that much for the purpose of this PR.

Reply:
I fully agree with using image from google_containers in deployment. I just don't like how it's the default repository in every Makefile here. 
In kubernetes/autoscaler we require user to set REGISTRY env variable or we fail the build. I think it's a better approach, since there is no good default (and google_containers is a particularly bad one, since you really don't want accidental pushes there).

Reply:
SGTM

Reply:
@bskiba pointed out to me that we actually use the same approach in kubernetes/kubernetes repo. So I guess it's consistent to do it here as well. Please disregard my earlier comment.

Reply:
Done.